en/releases/1.12.md

```diff
@@ -55,15 +55,16 @@ The release includes new hardware support for the following boards, peripherals,
 * ARK CAN Flow ([Read more about this product on the manufacturers site](https://arkelectron.com/product/ark-flow/))
 * mRo Ctrl Zero H7 (Experimental) ([Read more about this product on the manufacturers site](https://store.mrobotics.io/mRo-Control-Zero-F7-p/mro-ctrl-zero-f7.htm))
 
+These are removed:
+* Removal of discontinued Intel AeroFC
+
 ### MAVLink
 
 * **MAVLink Ethernet configuration ([PR#14460](https://github.com/PX4/PX4-Autopilot/pull/14460))**
   * MAVLink Ethernet channel settings such as UDP port, remote port and broadcast mode now can be changed dynamically via parameters.
-
-<!--
 * **Support for querying `COMPONENT_INFORMATION` ([PR#16039](https://github.com/PX4/PX4-Autopilot/pull/16039))**
+  * Parameter metadata is now automatically synchronized in combination with QGC daily.
```

Comment:
@bkueng Looks good, and I'm going to merge. Not what this might mean. I assume it means that the metadata goes to our server so that it can be downloaded by QGC if the vehicle supplies an HTTPS URL rather than an on-vehicle URL? If so, consider clarifying that.

Reply:
I'm comparing to the status quo where metadata is copied into the QGC tree.

Reply:
This is great then. I'd argue for a different wording if QGC released version supported component_information.  Thanks very much.